examples/07_joining_tables_with_FuzzyJoin.py

```diff
@@ -0,0 +1,160 @@
+"""
+Joining tables with FuzzyJoin
+================================
+
+In this example, we show how to join tables with the :function:`FuzzyJoin` function. 
+We also demonstrate why this method is the most easy and appropriate tool for handling 
+the joining of tables for users that want to improve their machine learning models quickly.
+We go through some of the rich options that allow for manipulation when joining tables,
+such as taking into account the precision of the match.
+
+To do so, we will predict the happiness score of a country from
+the 2022 [World Happiness Report](https://worldhappiness.report/).
+We will also use data provided from the [World Bank open data platform](https://data.worldbank.org/)
+in order to create a satisfying prediction model.
+
+"""
+
+###############################################################################
+# Data Importing and preprocessing
+# --------------------------------
+#
+# We import the happiness score data first:
+import pandas as pd
+
+df = pd.read_excel('https://happiness-report.s3.amazonaws.com/2022/Appendix_2_Data_for_Figure_2.1.xls')
+df.drop(df.tail(1).index, inplace = True)
+
+# Let's take a look at the table:
+df.head(3)
+ 
```

Comment:
```suggestion
 
 #################################################################
```
Add a break here, so that the output of the above line is shown right here